### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by StasOkhrym**

### CODE REVIEW

File: app/managers.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db3")
+        self.table_name = "actors"
+
+    def create(
```
Comment (mltdll) [inline app/managers.py:11]: I don't think it's an issue, but wouldn't this method's definition fit in 79 characters?

File: app/managers.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        self._connection.commit()
+
+    def all(self) -> list[Actor]:
+        actors_cursor = self._connection.execute(
+            f"SELECT id, first_name, last_name "
+            f"FROM {self.table_name}"
+        )
+        return [Actor(*row) for row in actors_cursor]
+
+    def update(
```
Comment (mltdll) [inline app/managers.py:31]: Same here.

**Review by mltdll — APPROVED** [review]

File: app/managers.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            (new_first_name, new_last_name, id_to_update)
+        )
+        self._connection.commit()
+
+    def delete(self, id_to_delete: int) -> None:
+        self._connection.execute(
+            f"DELETE FROM {self.table_name} "
+            f"WHERE id = ?",
+            (id_to_delete,)
+        )
```
Comment (Nattalli) [inline app/managers.py:50]: I think it would be better to add `self._connection.commit()` here too

**Review by Nattalli — CHANGES_REQUESTED** [review]

**Review by Nattalli — APPROVED** [review]
Well done!

**Pull request closed without merging**